Simple typo fixes in README (#598)
enthought/comtypes: README.md

`comtypes` is a lightweight pure Python [COM](https://learn.microsoft.com/en-us/windows/win32/com/component-object-model--com--portal) package based on the [`ctypes`](https://docs.python.org/library/ctypes.html) foreign function interface library.

`comtypes` allows you to define, call, and implement custom and dispatch-based COM interfaces in pure Python.

`comtypes` requires Windows and Python 3.7 or later.
- Version [1.2.1](https://pypi.org/project/comtypes/1.2.1/) is the last version to support Python 2.7 and 3.3–3.6.
- `comtypes` does not work with Python 3.7.6 and 3.8.1 as reported in [GH-202](https://github.com/enthought/comtypes/issues/202). This bug has been fixed in Python >= 3.7.7 and >= 3.8.2.
- Certain `comtypes` functions may not work correctly in Python 3.8 and 3.9 as reported in [GH-212](https://github.com/enthought/comtypes/issues/212). This bug has been fixed in Python >= 3.10.10 and >= 3.11.2.

## Installation

`comtypes` is available on [PyPI](https://pypi.org/project/comtypes) and can be installed with `pip`:
```sh
# PyPI
pip install comtypes
```

> The source code for `comtypes` may be found on [GitHub](https://github.com/enthought/comtypes).

Author: Looks good.

[![GitHub Repo stars](https://img.shields.io/github/stars/enthought/comtypes?style=social)](https://github.com/enthought/comtypes/stargazers) [![GitHub forks](https://img.shields.io/github/forks/enthought/comtypes?style=social)](https://github.com/enthought/comtypes/network/members)  
[![Tidelift Subscription](https://tidelift.com/badges/package/pypi/comtypes)](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=readme)

> `comtypes` is a lightweight pure Python [COM](https://learn.microsoft.com/en-us/windows/win32/com/component-object-model--com--portal) package based on the [`ctypes`](https://docs.python.org/library/ctypes.html) foreign function library.

Author: Agreed.

`comtypes` allows you to define, call, and implement custom and dispatch-based COM interfaces in pure Python.

`comtypes` requires Windows and Python 3.7 or later.
- Version [1.2.1](https://pypi.org/project/comtypes/1.2.1/) is the last version to support Python 2.7 and 3.3–3.6.
- `comtypes` does not work with Python 3.7.6 and 3.8.1 as reported in [GH-202](https://github.com/enthought/comtypes/issues/202), however, this bug has been fixed in Python 3.7 releases greater than or equal to 3.7.7 and Python 3.8 releases greater than or equal to 3.8.2.
- Certain `comtypes` functions may not work correctly in Python 3.8 and 3.9 as reported in [GH-212](https://github.com/enthought/comtypes/issues/212). This bug has been fixed in Python >= 3.10.10 and >= 3.11.2.

## Installation

> `comtypes` is available on [PyPI](https://pypi.org/project/comtypes) and may be installed with `pip`:

Reviewer: ```suggestion
`comtypes` is available on [PyPI](https://pypi.org/project/comtypes) and can be installed with `pip`:
```
If it is available on PyPI, it can be installed with `pip`. So I think 'can' is more appropriate than 'may'.
There are also other parts where 'will' or 'can' would be more suitable than 'may', so please consider them as well.

Author: Can and may are pretty interchangeable but I don't care much either way.

Reviewer: I think that using 'may' indicates a possibility, but using 'can' or 'will' emphasizes that the action is actually possible or will definitely occur.

For example, I think that "By installing A, you can use B" is more appropriate than "By installing A, you may use B" when describing software functionality.

It may be my own bias, but I think that when 'may' is used in software descriptions, it typically implies "not limited to this" or "there is uncertainty".

```sh
# PyPI
pip install comtypes
```

The source code for `comtypes` may be found on [GitHub](https://github.com/enthought/comtypes).

### Dependencies

`comtypes` is a pure Python package — it has no additional required dependencies.

Optional functionality may be enabled by installing:
- `numpy`, in order to process arrays as `numpy`'s `ndarray`.
- `mypy` or other static type checkers, in order to interpret type hints.

None of these packages, however, are required in order to run `comtypes`.

## Community

Issues may be tracked, bugs may be reported and contributions may be made to the codebase and documentation on `comtypes`' [GitHub](https://github.com/enthought/comtypes) repository.

> ## Enterprise
> 
> `comtypes` is available as part of the Tidelift Subscription. `comtypes` and the maintainers of thousands of other packages are working with Tidelift to deliver one enterprise subscription that covers all of the open source you use. To learn more, you can visit Tidelift's website [here](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=referral&utm_campaign=github).

Reviewer: ```suggestion
## For Enterprise

Available as part of the Tidelift Subscription.

This project and the maintainers of thousands of other packages are working with Tidelift to deliver one enterprise subscription that covers all of the open source you use.

[Learn more](https://tidelift.com/subscription/pkg/pypi-comtypes?utm_source=pypi-comtypes&utm_medium=referral&utm_campaign=github).
```

This section is quoted from [jaraco/tidelift](https://github.com/jaraco/tidelift).  
I think it is understandable as it is.

Author: Fair enough.
